pom.xml

```diff
@@ -6,39 +6,169 @@
   <groupId>net.mekomsolutions</groupId>
   <artifactId>eip-routes-senaite</artifactId>
   <version>1.0.0-SNAPSHOT</version>
-  <packaging>pom</packaging>
+  <packaging>jar</packaging>
 
   <name>Routes to support syncing between OpenMRS and SENAITE</name>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <camel-version>2.25.0</camel-version>
-  </properties>
-
-  <dependencies></dependencies>
-
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>package-routes</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
+  <properties>    
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
+      <camel-version>3.3.0</camel-version>
+   </properties>
+   <dependencyManagement>
+      <dependencies>
+         <!-- camel BOM -->
+         <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-parent</artifactId>
+            <version>${camel-version}</version>
+            <scope>import</scope>
+            <type>pom</type>
+         </dependency>
+      </dependencies>
+   </dependencyManagement>
+   <dependencies>
+   	  <dependency>
+   	  	<groupId>org.openmrs.eip</groupId>
+	    <artifactId>openmrs-watcher</artifactId>
+   	  	<version>1.0-SNAPSHOT</version>
+   	  	<scope>test</scope>
+   	  </dependency>
+      <dependency>
+         <groupId>junit</groupId>
+         <artifactId>junit</artifactId>
+         <version>4.13.1</version>
+      </dependency>
+      <!-- Camel -->
+      <dependency>
```

Comment:
Most of these camel-XXX dependencies are already transitively included by openmrs-watcher, so don't duplicate them by adding them again to your pom file

Reply:
Thanks. I've cleaned up the duplicate dependencies.